bin/update-releases.sh

```diff
@@ -27,7 +27,7 @@ declare -A MISSING_REASONS=(
 )
 
 declare -A REMAINING_REASONS=(
-    [src/scf-helper-release]="not upstream"
+    #[src/scf-helper-release]="not upstream"
```

Comment:
This should be deleted *if* we were keeping this script around.

However, I think we should delete this (and all other scripts related to local releases) completely since we no longer use them. They are just copies of the `SCF` scripts anyways, and will also continue to be in git history, so we can always get them back in the unlikely case that we need them again. But I expect that we'll try very hard not to have submoduled releases again, so let's keep things neat and tidy! 😄

Reply:
I wasn't sure if it was just a coincidence that we "run out" of submodules or a plan to not have any in the future. Removed all scripts (I think).